conda_smithy/ci_register.py

```diff
@@ -163,13 +163,17 @@ def appveyor_configure(user, project):
         raise ValueError(response)
     content = response.json()
     settings = content['settings']
-    skip_appveyor = u'skipBranchesWithoutAppveyorYml'
-    if not settings[skip_appveyor]:
-        print('{: <30}: Current setting for {} = {}.'
-              ''.format(project, skip_appveyor, settings[skip_appveyor]))
-    settings[skip_appveyor] = True
-    url = 'https://ci.appveyor.com/api/projects'.format(user, project)
+    for required_setting in (
+        u'skipBranchesWithoutAppveyorYml',
+        u'rollingBuildsOnlyForPullRequests',
+        u'rollingBuilds',
```

Comment:
Is it ok if we drop this line? Rolling builds on PRs seems fine. However on branches this could cancel releases, which seems problematic.

Reply:
Just the `rollingBuilds` option means both PRs and branches. Both `rollingBuilds` and `rollingBuildsOnlyForPullRequests` means only PRs